docs/chapters/03.txt

```diff
@@ -475,16 +472,26 @@ IEEE Binary Floats
 ==================
 
 According to the IEEE Standard for Binary Floating-Point Arithmetic, "single-"
-and "double-precision" numbers are defined as follows:
+and "double-precision" numbers are defined as follows.
 
-::
++-------------+----------+------------+-------------+----------------+-------------+
+| ::          | ::       | ::         | ::          | ::             | ::          |
+|             |          |            |             |                |             |
+|             |   SIGN   |   EXPONENT |   MANTISSA  |   EXPONENT     |   RANGE     |
+|   PRECISION |   [BITS] |   [BITS]*  |   [DECIMAL] |   [DECIMAL]    |   [DECIMAL] |
++=============+==========+============+=============+================+=============+
+| ::          | ::       | ::         | ::          | ::             | ::          |
```

Comment:
all of the cells in this row should be marked up as normal text, so you can drop the `::` line and re-indent them so that they are one " " space away from the left hand `|` line